config/rbac/csi_cephfs_ctrlplugin_role.yaml

```diff
@@ -1,7 +1,7 @@
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: csi-cephfs-ctrlplugin-r
+  name: csi-cephfs-ctrlplugin-role
```

Comment:
The `-r` suffix was chosen on purpose. Why change it?

Reply:
I couldn't deduce the intention based on https://github.com/ceph/ceph-csi-operator/blob/main/config/rbac/csi_cephfs_ctrlplugin_role_binding.yaml#L11

Reply:
@leelavg the changes are required in rolebinding not here. https://github.com/ceph/ceph-csi-operator/blob/70e00b8cda45cee32a6ad8239364c1f892a3a81c/config/rbac/csi_cephfs_ctrlplugin_cluster_role.yaml#L4C9-L4C33 is the name for the clusterrole.

Reply:
no, this is Role, not ClusterRole and anyways I'm refreshing PR to align w/ existing convention
```
sa is service account
rb is role binding
cr is cluster role
crb is cluster role binding
r is role
```